tests/x11/sshxterm.pm

```diff
@@ -15,20 +21,16 @@ sub run() {
     my $self = shift;
     mouse_hide(1);
     x11_start_program("xterm");
-    script_run("ssh -XC root\@localhost xterm");
-    type_string "yes\n";
-    wait_idle 6;
+    # (1) ss for wait until the socket is opened from localhost.
+    # "ESTAB", AND ::1:ssh, AND dport(destination-port) for control with exactitude, that a inc. port from ssh only from localhost is open.
+    script_run("for i in {1..100}; do /usr/sbin/ss -at dport = :ssh | tail -n1 >/dev/$serialdev; sleep 3; done & ");
```

Comment:
Can you please write your tests as humans would test? If you want headless unit tests, put them in scripts in /data. 

But this is explicitly a test checking xterm through ssh, *not* a headless test.